### PR TITLE
Adding parent header for tables

### PR DESCRIPTION
**Pull request by joshuahughes**
This branch extracts the new table column styles for bulk payroll and brings them into the Table component to make them available across the app.

### CODE REVIEW

**Comment by paddyduke** [comment]
This branch doesn't move the vertical border class down to the cell level, which we discussed last week. Is that something you're still planning to do (in a separate branch)?

File: assets/scss/global/components/_table.scss [hunk 1]
```diff
@@ -37,19 +37,19 @@ $Table-border-color: $color--gray-15;
...
   /* 3 */
-  margin: 0 0 1em;
+  margin: 0 0 1em 0;
   padding: 0;
   text-align: left;
   /* 4 */
   width: 100%;
+  @include font-weight(default);
+  @include line-height(4);
+  @include text-color(gray-5);
```
Comment (paddyduke) [inline assets/scss/global/components/_table.scss]: Our utilities should always come before the standard CSS rules.

http://localhost:9001/origin/guidance/code-style/#css-declaration-order

File: assets/scss/global/components/_table.scss [hunk 2]
```diff
@@ -112,17 +123,34 @@ $Table-border-color: $color--gray-15;
  */
 
 .Table-header--prominent {
+  /* 1 */
+  border-top: 1px solid white;
   @include background-color(gray-16);
```
Comment (paddyduke) [inline assets/scss/global/components/_table.scss]: Again, our utilities should come first.

File: assets/scss/global/components/_table.scss [hunk 3]
```diff
@@ -282,12 +306,37 @@ $Table-border-color: $color--gray-15;
...
+ * Add a right-hand border to an individual table cell.
+ */
+
+.Table .Table-cell--verticalBorder--right {
+  border-right: 1px solid $color--gray-13;
+}
+
+.Table .Table-cell--verticalBorder--header--right {
+  border-right: 1px solid desaturate(lighten($color--fa-blue, 45), 50);
+}
```
Comment (paddyduke) [inline assets/scss/global/components/_table.scss]: This looks good. Could you maybe add an example of the to the docs of using these with the `.Table-row--parentHeader`? Something similar to how you'd use them with the Bulk Payroll table?

**Comment by joshuahughes** [comment]
@paddyduke Added these examples for individual left/right cell borders to the documentation. Good call that :+1:
screen shot 2016-01-12 at 11 29 01

**Comment by paddyduke** [comment]
Looks great. PASS

**Pull request merged**